src/main/java/com/elmakers/mine/bukkit/action/builtin/CustomProjectileAction.java

```diff
@@ -649,8 +710,17 @@ else if (reorient)
         if (!block.getChunk().isLoaded()) {
             return miss();
         }
-
-        if (targetingResult == Targeting.TargetingResult.BLOCK) {
+        
+        if(distanceTravelled<min_range && targetingResult != null) {
+        	if(distanceTravelled>=min_block_range && targetingResult == Targeting.TargetingResult.BLOCK) {
```

Comment:
I'm a little confused by the >= here versus < ?

I'm hoping you can explain this to me, sorry! I thought I'd be able to comprehend it when I pulled but it took some scrutinizing to realize I'm not sure what all this is doing. :)